tests/integration/targets/auth_token/tasks/token_test_revocation.yml

```diff
@@ -0,0 +1,36 @@
+- name: "Test block"
+  module_defaults:
+    community.hashi_vault.vault_read:
+      url: '{{ ansible_hashi_vault_url }}'
+      auth_method: token
+  vars:
+    user_token: '{{ user_token_cmd.result.auth.client_token }}'
+  block:
+    # the token auth method never has ephemeral tokens, so we expect all tokens
+    # to continue to be usable even if `revoke_ephemeral_token` is set to true.
+    - name: Read token information using plugin
+      community.hashi_vault.vault_read:
```

Comment:
this is one of the reasons I generally avoid using the end-user facing plugins within the other integration tests, instead preferring the test plugins that are in `tests/integration/targets/setup_vault_test_plugins`, see `vault_ci_read`.